docs/company/reports/t1-2022/company-report-v2.md

```diff
@@ -690,21 +691,31 @@ Sarah Gosling
 
-#### *Aims for Trimester* 
+### What's been achieved so far
 
-The aim for this trimester is to work on game creation using SplashKit. This must incorporate all the functions that SplashKit has to make the games. This will showcase the creation tools that game developers can use to create a game from scratch so that it can be played on the arcade machine.
+For the game Jump, background and player both appear on the screen, image assets created on photoshop and exported as PNG files (as the pixel size wanted for display in the window to avoid scaling)
 
-*Deliverables* 
-Short-term:
+For the game Dicey Combat, background and one of the players (image assets) were created on Procreate. Created a complete plan for the game structure.
 
-- Cool 2D games demonstrating all functionality of SplashKit. (Minimum 1 per team member)
-- A step-by-step article guide of How to create the Cool games produced (Document design decisions, process and how-to-guides)
-- Guide uploaded to the SplashKit.io website.
+For the game Tanks, the A.I for the tanks has been created and the assets are being worked on. The current state of the game is sitting around 40% complete. The game class has been created also meaning that there is a working demo.
 
-Long-term:
+### Include a forecast of the likely state for each project at the end of the trimester
+
+As a team, we are expecting to be completing the game structure and at least halfway through programming the 2 games.
+
+For the Game Jump, I want to have at least 1 level able to be completed (finished moving to the next level) this includes player input, sounds, animations and speed, gravity, and velocity for movement. While progress to date has been slow, I remain optimistic.
+
+For the Game Dicey Combat, We want to be able to move both players across the board with the right direction movement. As well as having the score board synchronized with the players’ winning or losing points.
+
+For the Game Tanks, The game will most likely be finished at the end of this trimester and will be improved upon after the initial main game is finished.
+
+### Indicate any changes to plans for the project.
+
+The team's structure has shifted from four individual projects to three individual projects. This has allowed us to make more progress towards deliverables for the end of the trimester.
+
+### New projects that have started since 2.1P was created
 
-- Documentation updates on how to create cool games of different types
-  - create templates for platformers and collection games.
+The game Tanks was a new project that was added by Nick as a way to demonstrate A.I in splashkit. This isn't a neccessary project but one so that we can allow for more demonstration of SplashKit to be shown.
```

Comment:
```suggestion
The game Tanks was a new project that was added by Nick as a way to demonstrate A.I in splashkit. This isn't a necessary project but one so that we can allow for more demonstration of SplashKit to be shown.
```